wire/net/bus_test.go

```diff
@@ -19,7 +19,7 @@ import (
 
 	"github.com/stretchr/testify/assert"
 
-	"perun.network/go-perun/wire"
+	. "perun.network/go-perun/wire"
```

Comment:
We usually don't do this in go-perun. Is there a specific reason you use it here?

Reply:
it keeps the diff small.

Reply:
This was a lazy fix, I think I had copied some code from inside the wire package, and it took less typing to fix the type names that way. I re-did it now.